manual/development.md

```diff
@@ -197,7 +197,7 @@ module RuboCop
         def on_send(node)
           return unless not_empty_call?(node)
 
-          add_offense(node)
+          add_offense(node, :expression)
```

Comment:
Didn't we deprecate this a long time ago? You're supposed to do `add_offense(node, location: loc, message: msg)` now.